src/BottomNavigation/BottomNavigation.spec.js

```diff
@@ -0,0 +1,35 @@
+// @flow weak
+/* eslint-env mocha */
+
+import React from 'react';
+import { assert } from 'chai';
+import { createShallowWithContext } from 'test/utils';
+import BottomNavigation, { styleSheet } from './BottomNavigation';
+
+describe('<BottomNavigation>', () => {
```

Comment:
@oliviertassinari I don't think it should be a "if we have time" thing -- we **must** test the `selectedIndex` logic.